cpu/stm32_common/Makefile.include

```diff
@@ -8,5 +8,314 @@ USEMODULE += periph_common
 # include stm32 common functions and stm32 common periph drivers
 USEMODULE += stm32_common stm32_common_periph
 
+ROM_START_ADDR ?= 0x08000000
+RAM_START_ADDR ?= 0x20000000
+
+
+# The next block takes care of setting the rigth lengths of RAM and ROM
+# for the stm32 family. Most of the CPUs should be taken into account
+# here, so no need to add the variables per CPU or variant.
+STM32_INFO := $(shell printf '%s' $(CPU_MODEL) | tr 'a-z' 'A-Z' | sed -E -e 's/^STM32(F|L)(0|1|2|3|4|7)([0-9])([0-9])(.)(.)/\1 \2 \2\3\4 \3 \4 \5 \6/')
```

Comment:
I think it would make sense to split this into its own file, like kinetis-info.mk in cpu/kinetis

Reply:
True, I will adapt this to follow that schema. Thanks for the hint!